pkg/apis/networking/register.go

```diff
@@ -84,6 +84,19 @@ const (
 	// WildcardCertDomainLabelKey is the label key attached to a certificate to indicate the
 	// domain for which it was issued.
 	WildcardCertDomainLabelKey = "networking.knative.dev/wildcardDomain"
+
+	// CustomGatewayAnnotationKey is the annotation for specifying the Gateway the VirtualService
+	// should use. This will append to the list of Gateways that Knative creates. For example,
+	//
+	//    networking.knative.dev/customGateway: default/some-gateway
+	//
+	// The Gateway name format must be of either {namespace}/{gateway-name} or {gateway-name},
```

Comment:
why not ns.gw?

Reply:
oh i see its the format istio expects

Reply:
Yes Istio expects this qualified name format for gateways.